SPEC.md

```diff
@@ -77,7 +89,9 @@ Network configuration in JSON format is streamed to the plugin through stdin. Th
 
 ### Result
 
-Success is indicated by a return code of zero and the following JSON printed to stdout in the case of the ADD command. This should be the same output as was returned by the IPAM plugin (see [IP Allocation](#ip-allocation) for details).
+Success is indicated by a return code of zero and the following JSON printed to stdout in the case of the ADD command. This should be the same output as was returned by the IPAM plugin (see [IP Allocation](#ip-allocation) for details). GET command returns same output
+as ADD, without doing any additional configuration (in bridge plugin - GET command will enter network namespace and collect 
```

Comment:
Same as above. `e.x. in bridge plugin - GET command _should_ enter network namespace...`, but IMO this is still implementation detail which should not be so expressively seen in spec. IMO `bridge` plugin should not be treated here specially.